features/facebookImport/src/locales/en/miniStories/advertisingValue.json

```diff
@@ -1,5 +1,6 @@
 {
     "title": "Your advertising value",
     "intro": "Since Facebook makes its money by selling advertising they assign <strong>{{number_interests}}</strong> so-called advertising interests to you. For example, Facebook assumes you are interested in:",
-    "end.text": "Advertisers buy spots in your timeline according to those interests."
+    "end.text": "Advertisers buy spots in your timeline according to those interests.",
+    "details.text.1": "Facebook assigned the following <strong>{{number}}</strong> advertising interests to you."
```

Comment:
alt: "these" instead of "the following" Probably will need to change it elsewhere, too...

Reply:
Yes , the texts are just copy/pasted. Maybe I'll ask someone what would be the best procedere to review texts...

Reply:
Sven I suppose